README.md

```diff
@@ -31,21 +31,23 @@ jobs:
       with:
         creds: ${{ secrets.AZURE_CREDENTIALS }}
 
-    - name: 'Sync NGINX configuration to NGINX on Azure instance'
+    - name: 'Sync the NGINX configuration from the Git repository to the NGINX for Azure deployment'
```

Comment:
Thanks for making this change to consistently use "NGINX for Azure" within the docs and code.

Reply:
Of course! Will do a final description and documentation polishing after this PR and before publishing the action.